Makefile

```
@@ -288,4 +288,4 @@ build/libprotobuf-mutator:
 	$(MAKE) -C build/libprotobuf-mutator
 
 # picks up include dependencies for all object files
--include $(OBJECTS:.o=.d)
+-include $(OBJECTS:.o=.d)
```

```suggestion
-include $(OBJECTS:.o=.d)

```

Don't remove trailing newlines in files.